pkg/util/image_prep_utils.go

```diff
@@ -87,13 +87,23 @@ func getFileSystemFromReference(ref types.ImageReference, imageName string) (str
 	for _, b := range img.LayerInfos() {
 		bi, _, err := imgSrc.GetBlob(b)
 		if err != nil {
-			glog.Errorf("Failed to pull image layer with error: %s", err)
+			glog.Errorf("Failed to pull image layer: %s", err)
```

Comment:
We should probably return here right? This will continue. I realize it was like this before, but we should clean it up while you're changing the line.

Reply:
Ah yeah, don't think it makes sense to keep going here cause this totally screws up the image representation. Fixed